scss/components/_tola_management.scss

```diff
@@ -151,6 +152,12 @@
     }
 }
 
+.disagg-form__fieldset:disabled a.btn.btn-link.btn-danger {
```

Comment:
TIL you can select on disabled in CSS... this obviates so many helper classes we have elsewhere